examples/plots/plot_interpolation_for_transform_manager.py

```diff
@@ -0,0 +1,101 @@
+"""
+======================
```

Comment:
The length of the `=`s has to match the length of the title.

Reply:
fixed